wiki/History_of_osu!/2009/de.md

```diff
@@ -37,3 +31,128 @@ osu! gelangt durch eine Portierung von nuudles auf das iPhone. Das Projekt bekam
+
+## Februar
+
+Am 14. Februar wurde die vierte Austragung des osu! Community Meetings abgehalten, bei dem es um die Moderation von Beatmaps, eine neue, kommende Übersetzungsfunktion und Fragen bezüglich der Erweiterung des osu!-Teams ging.[^community-meeting-4]
+
+Das [Kudosu-System](/wiki/Kudosu) wurde zur Vereinfachung des [Beatmap-Modding](/wiki/Modding) im [Ranking-Verfahren](/wiki/Beatmap_ranking_procedure) implementiert. Mapper konnten Leute mit Punkten (genannt Kudosu) für nützliche und konstruktive Kritiken honorieren. 3 Kudosu-Punkte konnten gegen 1 Kudosu-Stern eingetauscht werden, der dann dazu genutzt werden konnte, um den Rang der Beatmap in der Modding-Prioritätenliste des [Beatmap Appreciation Teams](/wiki/People/Beatmap_Appreciation_Team) (BAT) zu erhöhen.[^kudosu-1]
+
+## März
+
+Drei Aktualisierungen diesen Monat sorgten für viele Änderungen in osu!. Die erste Änderung bestand aus Filteroptionen für Mehrspieler-Lobbies sowie einer Chat-Einstellung zum Ausblenden anstößiger Sprache. Durch diese Einstellung wurde jedes unzulässige Wort gegen ein Wort, das in einer Liste vorgegeben wird, ersetzt.[^stable-b622]
+
+Im darauffolgenden Update am 21. März wurde [Kiai-Zeit](/wiki/Gameplay/Kiai_time) hinzugefügt, um Refrains oder bedeutende Teile eines Songs mit Sternfontänen betonen zu können.[^stable-b631]
+
+Am 26. März erhielt osu! Unterstützung für den TaTaCon Wii Taiko Drum Controller sowie Unterstützung für die Steuerung mit einem Wiimote (nur in Catch The Beat).[^stable-b639][^custom-changelog]
+
+## April
+
+Die Zahl der registrierten Nutzer überstieg 100.000.[^user-count]
+
+## Mai
+
+Eine Veränderung im Kudosu-System erlaubte Nutzern Kudosu-Sterne an jede Beatmap zu verteilen, nicht nur an ihre eigenen wie zuvor.[^kudosu-change]
+
+Neben anderen Neuerungen erhielt osu! in einem Update diesen Monat Breitbildunterstützung und die Mod [Autopilot](/wiki/Gameplay/Game_modifier/Autopilot). Mit der Mod wurde der Cursor automatisch bewegt und der Spieler musste nur das Tippen übernehmen.[^stable-b674]
+
+## Juni
+
+Durch eine Umstellung im [Beatmap-Einreichungssystem](/wiki/Beatmapping/Beatmap_submission) konnten Nutzer keine Änderungen mehr an Beatmaps machen, nachdem diese gemoddet waren. Es gab Fälle, in denen Mitglieder des BAT Beatmaps gerankt haben und ihnen nicht bewusst war, dass die Beatmaps [regelbrechende Modifikationen](/wiki/Ranking_criteria) enthielten.[^bss-update]
+
+Bis zu diesem Zeitpunkt war osu! nur in Englisch verfügbar. Mit der Einführung des neuen Lokalisierungssystems bekamen Nutzer die Möglichkeit, den Spielclient zu übersetzen. Sie konnten eine Datei mit den Textzeilen des Spiels herunterladen, den Text in ihre Sprache übersetzen und dann die Datei vom Entwicklerteam in das Spiel integrieren lassen.[^client-localisation][^stable-b695]
```

Comment:
It would be better to keep the translation for `localisation` (feature/system/etc.) consistent with the February paragraph. I don't mind either one, but it would be good to have a connection between the two (as in the English version).

Maybe `Lokalisierungsfunktion` above.. I'll let you figure it out